spec/router_spec.rb

```diff
@@ -0,0 +1,22 @@
+RSpec.describe Router do
+  def app
+    Rack::Builder.parse_file('config.ru').first
+  end
+
+  context 'when no game' do
+    it 'index page works well' do
+      get(Constants::HOME_PATH)
+      expect(last_response.body).to include('Codebreaker')
+    end
+
+    it 'statistics page works well' do
+      get(Constants::STATISTICS_PATH)
+      expect(last_response.body).to include('Top of Players:')
```

Comment:
Instead of string it would be better to use I18n, constant or value from let variable

Reply:
+